tests/lib/reset.sh

```diff
@@ -74,7 +74,12 @@ reset_classic() {
         systemctl start snap.mount.service
     fi
 
-    rm -rf /root/.snap/gnupg
+    # Clean root home
+    rm -rf /root/snap /root/.snap/gnupg /root/.{bash_history,local,cache,config}
+    # Clean test home
+    rm -rf /home/test/snap /home/test/.{bash_history,local,cache,config}
+
+    rm -rf /snap/*
```

Comment:
purge should have done it, perhaps there should be a `TODO` to inspect if there's anything left behind in the /snap directory

Reply:
Ok, I'll remove this nad check to see which files are being not deleted in /snap, then I can restore this and leave a comment with that info